docs/reference/seismogram.rst

```diff
@@ -18,31 +18,33 @@ Properties
 .. autosummary::
    :toctree: api/
 
-    Seismogram.events
-    Seismogram.events_columns
-    Seismogram.target_event
-    Seismogram.target
-    Seismogram.time_zero
-    Seismogram.output
-    Seismogram.dataframe
-    Seismogram.output_path
-    Seismogram.censor_threshold
-    Seismogram.prediction_count
-    Seismogram.entity_count
-    Seismogram.feature_count
-    Seismogram.start_time
-    Seismogram.end_time
-    Seismogram.event_count
-    Seismogram.event_types_count
-    Seismogram.cohort_attribute_count
-    Seismogram.score_bins
-    Seismogram.data
+   Seismogram.events
```

Comment:
Why the change by one space?

Reply:
I was trying to set up my IDE to identify restructuredtext for basic autoformatting (primarily tabsize and trailing whitespace) and the main style guides I found said 3-space tabs for rst though it doesn't seem fully established.  
It seems like ours is _mostly_ 3, but some interspersed 4-space.

Reply:
Is there to formalize this as an expectation?